CONTRIBUTING.md

```diff
@@ -10,3 +10,14 @@ Contributions are welcome, but please review this document before making any cha
 * Use Promises API. Avoid blocking/sync calls.
 * Use logging framework. Avoid `console.log()` and `console.dir()`.
 * Avoid unecessary dependencies to 3rd party modules.
+
+## Pull Requests
+* We use the [Forking process](https://guides.github.com/activities/forking/) in our development. [Here](https://blog.scottlowe.org/2015/01/27/using-fork-branch-git-workflow/) are more details and best practices
+* Follow our Lining rules
+* Maintain quality
+
+## Git Commit Messages styleguide
+* Use the present tense ("Add feature" not "Added feature")
+* Use the imperative mood ("Move cursor to..." not "Moves cursor to...")
+* Limit the first line to 72 characters or less
```

Comment:
Maybe an elaboration on what this means or maybe a discussion with team on what this means. I was also thinking it might be a good idea to use a template in the description while creating a PR. This is what I use:
Brief Description:
What has been added:
What has been changed:
What has been removed: